platforms/access-bridge/README.md

```diff
@@ -62,6 +62,33 @@ You can also copy and paste the contents of `.env.example` into `.env.local` and
   }
   ```
 
+#### URL: `/v2/sites/{site_id}/products`
+
+- **Method:** GET
+- **Authorization:** None
+- **Summary:** Lists all the corresponding stripe products with prices that are connected to the SIMS plans.
+- **Response:** Stripe.Product type
```

Comment:
Maybe it's worth it to include a link to the latest stripe format doc and re-emphasize that this is just a passthrough of what stripe tells us